Classes/Domain/Model/Category.php

```diff
@@ -29,18 +29,15 @@
  *  GNU General Public License for more details.
  *
  *  This copyright notice MUST APPEAR in all copies of the script!
- ***************************************************************/
+ */
 
 /**
- *
- *
- * @package pxa_product_manager
  * @license http://www.gnu.org/licenses/gpl.html GNU General Public License, version 3 or later
- *
  */
 class Category extends CategoryExtbase
 {
-    use AbleCacheProperties, CanCreateCollection;
+    use AbleCacheProperties;
```

Comment:
I suggest we refactor this and all other Able* namespaces to Can* now. It's better English.

```suggestion
    use CanCacheProperties;
```